### ISSUE TITLE
Move test_env up to the Test Suite level

### ISSUE THREAD

**Issue by maximusunc** [NCATSTranslator/Tests#14]
Is there ever a case where different tests in a single test suite would hit different environments? Or is a test case for a specific environment? Or would we ever run the same test against multiple environments all in the same Test Suite? I'm not seeing these types of use cases, so it seems to make sense to move this field up to the Test Suite level.

**Comment by maximusunc** [comment]
I'd like to bump this priority. Given the Test Plan is nearing enactment, I see each Test Suite as being specific to an environment, and the Test Suites are going to be using the same Test Cases but in different environments potentially.

**Comment by RichardBruskiewich** [comment]
Hi @maximusunc, I generally concur. I've never run any test suites (in Translator test framework pre-history) against more than one target server at a time, which by definition, is running inside a specified environment.

I might even be tempted to argue that the specification of test environment lies one level above the TestSuite, opening up the prospect of the same given TestSuite being run against more than one environment?

A TestSuite here seems like a coherent collection of TestCases (or TestAssets bundled with procedural directions guiding their conversion into TestCases, unless, of course, that's simply embedded inside the TestRunners - I suspect they are...)

**Comment by maximusunc** [comment]
I agree that it might make sense to put the test env even one higher, but given that we don't have any more levels, I think it's fine to potentially duplicate Test Suites that run in different environments.

**Comment by RichardBruskiewich** [comment]
I wonder if a simple composition data model for a test run itself might resolve this: there are likely several orthogonal components that define a test run: the test suite, environment, maybe other metadata? @sierra-moxon what do you think?
